guides/source/getting_started.md

```diff
@@ -334,216 +340,650 @@ If we go back to our browser and make a request to
+
+```ruby
+irb(main):005:0> Article.create(title: "Post #2", body: "Still riding the Rails!")
+```
+
+This way, we don't need to call `new` and then `save`.
+
+Lastly, models provide a method to find all of their data:
+
+```ruby
+irb(main):006:0> articles = Article.all
+ #<ActiveRecord::Relation [
+   #<Article id: 1, title: "Hello Rails", body: "I am on Rails!", created_at: "2020-01-18 23:47:30", updated_at: "2020-01-18 23:47:30">,
+   #<Article id: 2, title: "Post #2", body: "Still riding the Rails!", created_at: "2020-01-18 23:53:45", updated_at: "2020-01-18 23:53:45">]>
+```
+
+This method returns an `ActiveRecord::Relation` object, which you can think of
+as a super-powered array. This array contains both of the topics that we have
+created so far.
+
+TIP: For more about models, read the [Active Record
+Basics](https://guides.rubyonrails.org/active_record_basics.html) and [Active
+Record Query
+Interface](https://guides.rubyonrails.org/active_record_querying.html) guides.
+
+As you can see, models are very helpful classes for interacting with databases
+within Rails applications. Models are the final piece of the "MVC" puzzle.
+Let's look at how we can go about connecting all these pieces together into a
+cohesive whole.
+
+### Viewing a List of Articles
+
+Now that you've seen how to create a route, a controller, an action, a view and
+a model, let's connect these pieces together.
+
+Let's go back to `app/controllers/articles_controller.rb` now. We're going to
+change the `index` action here to use our model.
 
 ```ruby
 class ArticlesController < ApplicationController
+  def index
+    @articles = Article.all
+  end
 end
 ```
 
-A controller is simply a class that is defined to inherit from
-`ApplicationController`.
-It's inside this class that you'll define methods that will become the actions
-for this controller. These actions will perform CRUD operations on the articles
-within our system.
+Controller actions are where we assemble all the data that will later be
+displayed in the _view_. In this `index` action, we're calling `Article.all`
+which will make a query to our database and retrieve all of the articles,
+storing them in an instance variable: `@articles`.
+
+We're using an instance variable here for a very good reason: instance
+variables are automatically shared from controllers into views. So to use this
+`@articles` variable in our view to show all the articles, we can write this
+code in `app/views/articles/index.html.erb`:
+
+```erb
+<h1>Articles</h1>
+
+<ul>
+  <% @articles.each do |article| %>
+    <li><%= article.title %></li>
+  <% end %>
+</ul>
+```
 
-NOTE: There are `public`, `private` and `protected` methods in Ruby,
-but only `public` methods can be actions for controllers.
-For more details check out [Programming Ruby](https://ruby-doc.org/docs/ProgrammingRuby/).
+We've now changed this file from using just HTML to using HTML and _ERB_. ERB
+is a language that we can use to run Ruby code.
```

Comment:
I don't want to split hairs here but I think it's confusing calling ERB a language when it ships with Ruby. Ruby calls it a templating system https://ruby-doc.org/stdlib-2.6.5/libdoc/erb/rdoc/ERB.html

```suggestion
is a templating system that we can use to run Ruby code.
```